Mammoth/TSE/CParticleArray.cpp

```diff
@@ -1549,9 +1549,30 @@ void CParticleArray::ReadFromStream (SLoadCtx &Ctx)
 	
 	//	Previous version didn't have everything
 
-	if (Ctx.dwVersion >= 120)
+	if (Ctx.dwVersion >= 212)
 		Ctx.pStream->Read((char *)m_pArray, sizeof(SParticle) * m_iCount);
 
+	else if (Ctx.dwVersion >= 119)
```

Comment:
I think this should be: >= 120

Right? From 120 to 211 we're using the SParticle212 structure (which maybe should be renamed to SParticle120?)